class/class-feature.php

```diff
@@ -0,0 +1,261 @@
+<?php
+/**
+ * @package    Feature
+ */
+
+/**
+ * Feature
+ */
+class Feature {
+
+	/**
+	 * Feature list
+	 */
+	public static function feature_list() {
+
+		$all_features = Database::get_all_feature_data();
+		if ( ! empty( $all_features ) ) {
+			self::display_features( $all_features );
+		} else {
+			echo 'NO DATA';
+		}
+	}
+
+	/**
+	 * Display features
+	 *
+	 * @param array $all_features it have details of features.
+	 */
+	public static function display_features( $all_features ) {
+
+		foreach ( $all_features as $feature ) {
+			?>
+				<div class="wrapper mt-5 mb-5 pb-3 row">
+
+					<div class="vote col-2 p-4">
+						<?php self::feature_vote( $feature->id ); ?>
+						
+					</div>
+
+					<div class="col-8">
+						<h4  class="feature-head" id="<?php echo esc_html( $feature->id ); ?>" onclick="edit_feature(event,<?php echo esc_html( $feature->id ); ?>)"><?php echo esc_html( $feature->title ); ?></h4>
```

Comment:
For all attributes like id, onclick it will be better to add esc_attr() function instead of esc_html()

Reply:
Okay